File: Strings/Word.go
package main

import (
	"fmt"
	"strings"
	"unicode"
)

func main() {
	var str string
	fmt.Scanln(&str)
	var upper, lower int
	for _, ch := range(str) {
		if unicode.IsLower(ch) {
			lower++
		} else {
			upper++
		}
	}

	if upper > lower  {
		str = strings.ToUpper(str)
	} else {
		str = strings.ToLower(str)
	}
	fmt.Println(str)
	
	
}

File: Strings/Word.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main() {
    string str;
    cin >> str;
    int upper = 0;
    int lower = 0;

    for( auto ch : str) {
	if( ch >= 'A' && ch <= 'Z')  upper++;
	else lower++;
    }
    if( upper > lower )
	transform( str.begin(), str.end(), str.begin(), ::toupper);
    else
	transform(str.begin(), str.end(), str.begin(), ::tolower);

    cout << str << "\n";
			  
}
    
